### DISCUSSION TITLE
v0.4.12

### DISCUSSION THREAD

**Discussion by DrB-S** [DrB-S/heatcluster#11] [Announcements]
Fully functional version, properly implementing all github actions.

<hr /><em>This discussion was created from the release <a href='https://github.com/DrB-S/heatcluster/releases/tag/v0.4.12'>v0.4.12</a>.</em>

**Comment by DrB-S** [comment]
Does the zip file contain what is necessary for the docker image?

**Comment by erinyoung** [comment]
Yes, although the source code also contains what is necessary. When I'm done eating lunch I'll create a Dockerfile to be hosted by UPHL so can get the gist of things.
